src/dwarf.cpp

```diff
@@ -1366,6 +1366,28 @@ QString Dwarf::happiness_name(DWARF_HAPPINESS happiness) {
     }
 }
 
+QString Dwarf::get_all_goal_names() {
```

Comment:
Make the method const if possible. I know it is not done properly everywhere and it can ripple through calls. But new method should try to be correct.

Reply:
Changed to const